### ISSUE TITLE
Make a generic components for incident and report cards

### ISSUE THREAD

**Issue by lmcnulty** [responsible-ai-collaborative/aiid#956]
We currently have a bunch of different places where we display incidents or reports in cards. These all have different styles:

<img src="https://user-images.githubusercontent.com/25443411/185217536-a7e8d71a-a0a4-4f74-a9ab-318ef37aacfc.png" height="300"/> <img src="https://user-images.githubusercontent.com/25443411/185217725-04e7d0b7-2308-42cd-8881-920d934c5a0a.png" height="300" /> <img src="https://user-images.githubusercontent.com/25443411/185218008-74155edd-3747-40d8-86e2-9065d40d434f.png" height="300"/>

![image](https://user-images.githubusercontent.com/25443411/185217599-accf910f-f441-4ac1-bc78-219a7f6382f0.png)

![image](https://user-images.githubusercontent.com/25443411/185218278-3bddd6d8-4492-47d1-8f24-f2ac7657298f.png)

We should convert these into a single generic component that can render appropriately in response to parameters. This should go along with moving to Tailwind.

**Comment by smcgregor** [comment]
@clari182 I think you may be working this one in another way?

**Comment by clari182** [comment]
@smcgregor Yes. Now we are using Flowbite, so any time we need a card we use their card's component. There are exceptions though, that can be improved, like the sections' cards on the citation page.
I can take this issue and work on improvements where possible!
